internal/commands/artifact_import_test.go

```diff
@@ -0,0 +1,44 @@
+package commands_test
+
+// func TestArtifactImport(t *testing.T) {
+// 	responses := []interface{}{
+// 		gqlmock.MockQueryResponse("getArtifactDefinitions", []api.ArtifactDefinitionWithSchema{
```

Comment:
I couldn't get this mock to work. I think its partly because the response is an array instead of a struct, and I didn't see any existing examples to copy.